Cargo.toml

```diff
@@ -29,7 +29,7 @@ debug = true
 
 [dependencies]
 fixedbitset = { version = "0.1.4" }
-quickcheck = { optional = true, version = "0.8", default-features = false }
+quickcheck = { optional = true, version = "0.8.5", default-features = false }
```

Comment:
Does this need to be 0.8.5? or will any 0.8 work?

Reply:
Yes, 0.8 will work too. I just saw that there was a quickcheck version bump, but not to the latest one. I don't see why not to bump to the latest quickcheck for the new release though. You can drop this commit if you don't want to.

Reply:
Okay. If you can drop that commit from this PR, we can get this landed.